src/components/CodingFetch.js

```diff
@@ -31,19 +31,20 @@ function CodingFetch() {
         <section className="post-card">
           <div className="post-details">
             <div className="post-name-desc">
-              <p className="post-title">TITLE:{post.title}</p>
-              <p className="post-body">BODY:{post.body}</p>
+              <p className="post-title">TITLE:</p>
+              {post.title}
+              <p className="post-body">BODY:</p>
+              {post.body}
+              <p className="post-topic">#TOPIC </p>
+              #{post.topic}
+              <p className="post-comments">COMMENT COUNT: </p>
+              {post.comment_count}
+              <p className="post-author">WRITTEN BY:</p>@{post.author}
+              <p className="post-created-at">CREATED AT:</p>
+              {post.created_at}
+              <p className="post-vote-count">VOTE COUNT:</p>
+              {post.votes}
```

Comment:
everything not in a `<p>` is just a text node - this is a bit unusual!

it would be best to wrap each thing in it's own tag